src/test/javascript/spec/component/overview/course-conversations/dialogs/forward-messages-dialog/forward-messages-dialog.component.spec.ts

```
@@ -0,0 +1,224 @@
+        component.channels.set([{ id: 1, name: 'General' } as ChannelDTO, { id: 2, name: 'Announcements' } as ChannelDTO]);
+        component.users.set([
+            {
+                id: 3,
+                name: 'User1',
+                imageUrl: 'user1.png',
+            } as UserPublicInfoDTO,
+        ]);
+        component.postToForward.set({ id: 10, content: 'Test Message', author: { id: 1, name: 'Author', imageUrl: 'author.png' } } as Post);
+
+        fixture.detectChanges();
+    });
```

Biome (1.9.4)

[error] Disallow duplicate setup and teardown hooks.

Disallow beforeEach duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)